src/overlays/actors/ovl_En_Nutsball/z_en_nutsball.h

```diff
@@ -7,7 +7,9 @@ struct EnNutsball;
 
 typedef struct EnNutsball {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x50];
+    /* 0x144 */ s16 timer;
+    /* 0x146 */ s16 startTimeOC;
```

Comment:
Maybe `timerThreshold` instead? This is a value checked against the timer that says when collision should activate.